bigquery_etl/query_scheduling/templates/public_data_json_airflow_dag.j2

```diff
@@ -45,6 +45,9 @@ with DAG('{{ name }}', default_args=default_args{%+ if schedule_interval != None
         + ["--parameter={{ task.date_partition_parameter }}:DATE:{% raw %}{{ds}}{% endraw -%}"]
         {%- endif -%},
         image=docker_image,
+        sla=datetime.timedelta(minutes=5),
+        execution_timeout=datetime.timedelta(minutes=10),
+        email_on_retry=False,
```

Comment:
I'm only changing the `bqetl_public_data_json` DAG here for testing. If everything works out we can update the remaining DAGs

Reply:
+1 to this plan.